Gruntfile.js

```diff
@@ -65,8 +64,14 @@ module.exports = function(grunt) {
           'modules/reliability/js/columnFormatter.js': 'modules/reliability/jsx/columnFormatter.js'
         }
       }
+    },
+
+    run: {
+      webpack: {
+        cmd: 'webpack'
```

Comment:
When running webpack from terminal, everything works, but when running with grunt, I get this error:
```bash
Running "run:webpack" (run) task
Verifying property run.webpack exists in config...OK
File: [no files]
Options: wait, failOnError=false, quite=false, ready=1000, cwd="/var/www/loris", passArgs=[], itterable=false, readyBufferLength=1024
running webpack with args []
>> Error: spawn webpack ENOENT
Warning: non-zero exit code -2 Use --force to continue.

Aborted due to warnings.
```